Machine_Learning/Machine_Learning.md

```
@@ -0,0 +1,122 @@
+# Beginner's Guide to Machine Learning
+
+<div align="center">
+<img src="machine-learning.jpg"  width="80%" loading="lazy">
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)